README.md

```diff
@@ -1,57 +1,202 @@
-# Project Name
+<!--
+---
+description: This Python sample shows how to use Durable Functions for model evaluation. 
+page_type: sample
+products:
+- azure-functions
+languages:
+- python
+- bicep
+- azdeveloper
+---
+-->
 
-(short, 1-3 sentenced, description of the project)
+# Model evaluation using Durable Functions (Python)
 
-## Features
+## About sample
 
-This project framework provides the following features:
+This sample demonstrates how to use Durable Functions to call multiple models in parallel to quickly get the best response to a user's query. It uses three models - GPT-3.5-turbo, GPT-4o-mini, and Phi-4 - to answer a query. After getting the responses, it uses GPT-4 to evaluate and score the responses based on a certain criteria. 
 
-* Feature 1
-* Feature 2
-* ...
+![Screenshot of sample-architecture](./media/sample-architecture.png)
 
-## Getting Started
+There's no particular reason for choosing the models used in this sample - the key is to demonstrate how to leverage Durable Function's fan-out/fan-in pattern to easily realize this scenario. 
 
-### Prerequisites
+### About Durable Functions 
+
+[Durable Functions](https://learn.microsoft.com/azure/azure-functions/durable/durable-functions-overview) is part of [Azure Functions](https://learn.microsoft.com/azure/azure-functions/functions-overview) offering. It helps  orchestrate stateful logic that is long-running and provides reliable execution. For example, when there's infrastructure failure (network connectivity dropped, VM crashed, etc.), the framework rebuilds application state and start from the point of failure instead of the beginning. This helps save time and money, especially for expensive operations like LLM calls. Common scenarios where Durable Functions is useful include agentic workflows, data processing, asynchronous APIs, batch processing, and infrastructure management.
```

Comment:
The network connectivity dropped example isn't quite right. You won't lose in-memory state because of this. I would use "process crash" and "VM restart" as two examples where we rebuild start and start from the point of failure.